### DISCUSSION TITLE
Set background color and texture

### DISCUSSION THREAD

**Discussion by BlockOG** [pokepetter/ursina#224] [Q&A]
I want to have a background but can't because there's no way to set the background color or texture
I tried making a big plane and setting it a color and it worked, but there's no way to make a texture that repeats since the plane is 10000x10000
So, if there's a way to set the background color then tell me (and texture too)
And if there's a way to make a repeated texture but not the said aboe then tell me that

**Comment by AshishPandagre** [comment]
![Screenshot (146)](https://user-images.githubusercontent.com/53047854/128600556-5f0c8fac-5796-4390-b38f-5ab594afe358.png)

I guess you are asking for something like this. Here's the code :

`ground = Entity(model='plane', scale=45, texture='textures/crate', texture_scale=(45,45), collider='box', color=color.red)`

**Comment by PixelTomato** [comment]
I interpreted your question as asking for a way to create a custom sky. To create a skybox, you can use the built-in `Sky()` class:

```py
from ursina import *

app = Ursina()

skybox_image = load_texture("custom_skybox_image.png")
Sky(texture=skybox_image)

app.run()
```

This script will load a new texture from whatever path your enter in `load_texture()` and display it on the skybox which will create a background to your level that will be "infinite" in size. You can also set the color of the sky by using `color=color.blue` or whatever other color you would like.

**Reply by Donyeet** [reply]
I tried this code,
```
skybox_image = load_texture("imgs\hell_skybox.jpg")
Sky(texture=skybox_image)
```
but the original skybox is still there. Do you know how to delete the original sky box?
